lib/config/executor/container_vm.go

```diff
@@ -143,6 +143,9 @@ type ExecutorConfig struct {
 
 	// version
 	Version *version.Build `vic:"0.1" scope:"read-only" key:"version"`
+
+	// migrated - this is used during in memory migration to assign whether an execConfig is viable for a commit phase
+	Migrated bool
```

Comment:
@emlin why not here in the container base instead? https://github.com/matthewavery/vic/blob/d5c7717969793557f357f1ba969d17bf14669755/lib/portlayer/exec/base.go#L53-L63 

ince we want to perform the migration close to the `decode` call.